src/utils/MMECalculator/OMTKData.js

```diff
@@ -0,0 +1,8560 @@
+// Original source: ./src/cql/OMTKData.cql
```

Comment:
How was this file generated from the original source? Is it a repeatable process that we could document?

Reply:
@ivan-c The [documentation](https://www.fhir.org/guides/cdc/opioid-mme-r4/Library-OMTKData.html) is not entirely clear on how it was generated (the link to how the file was generated is broken).  But the documentation referenced RxNav as the source from where the data originate.  So we will need to figure out how to get the data from RxNav eventually if we are going to update this file.